src/sample_submission_data.R

```diff
@@ -0,0 +1,88 @@
+## Script that generates sample submission data to pass validation checks for
+## the forecasting hub
+## It's not particularly efficient but it's easy to understand
+
+## Function to generate simulated data
+# Uses the Dirichlet distribution to generate random proportions of clades
+require(brms) # for data simulation
```

Comment:
suggest putting these require statements within the function call. related to my comment below about having one file for the function and one for the script.